docs/rest.md

```diff
@@ -247,6 +247,14 @@ Which channel to search in.
    derived by Geo IP lookup.
 
 ## /v2/snaps
+
+### Parameters:
+
+#### `select`
+
+Filter from the given selection. Currently only limiting to refreshable
+snaps is supported via the `refresh` key.
```

Comment:
Another motivator for this: `select=refresh` is a bit of a misnomer. It's not filtering.. it's changing the result set completely.

As an alternative for the CLI if `snap find --refresh` sounds awkward, we might go full circle into your proposal of `snap refresh --list`. Internally, that would might still map to `cli.Find` per idea above. This might be sensible as the output would look more like `list` than `find`, and killing the `--list` option exercises what was just exposed.

Apologies again. I realize this has seen a long process already, and I feel bad for discussing it further, but it's probably wise to spend a bit more time on this if we feel it's worth it.

What do you think? @chipaca? @pedronis?